src/app/sidebar/components/Card.tsx

```diff
@@ -25,9 +25,47 @@ export interface CardProps {
    * Boolean to check Card is selected by user
    */
   selected?: boolean;
+
+  /**
+   * Boolean to check if in schedule mode
+   */
+  schedule?: boolean;
 }
 
-export function Card({ subject, title, code, selected }: PropsWithChildren<CardProps>): JSX.Element {
+export function Card({ subject, title, code, selected, schedule }: PropsWithChildren<CardProps>): JSX.Element {
+  const buttons = (code: string | undefined, schedule: boolean | undefined) => {
+    if (!code) {
+      return (
+        <Box>
+          <IconButton
+            aria-label="Select course"
+            icon={<ChevronRightIcon />}
+            size="md"
+            background="null"
+            _hover={{ bg: 'none' }}
+          />
+        </Box>
+      );
+    } else if (code && schedule) {
+      return (
+        <VStack spacing="-0">
+          <IconButton
```

Comment:
I think the buttons should be consistent with [how Joey styled them](https://discord.com/channels/754490734622802075/798385368096833566/837205375433703484)